apps/events/tests/events/tests_events_get.py

```diff
@@ -76,9 +77,10 @@ def test_event_get_authorised_valid(
         'tags': list(event.tags.all()),
         'category': event.category,
         'created_by': event.created_by,
-        'location': (event.location.x, event.location.y),
+        'city_location': None,
```

Comment:
we must keep event location as is in the event. It is main field for each event, without it we cannot show were event is located. Location stays the same. We add new field city to event which will hold city location, place_id, and borders